README.md

```diff
@@ -268,6 +268,9 @@ Nat. Commun. 13:1024, 2022.
 - <a name="Zou22mir"></a>Zou, XL., Huang, T.J., Wu, S.,  
 *[Towards a New Paradigm for Brain-inspired Computer Vision](https://doi.org/10.1007/s11633-022-1370-z)*,  
 Mach. Intell. Res., 19:412-424, 2022.
+- <a name="gehrig22arxiv"></a>D. Gehrig, D. Scaramuzza,  
```

Comment:
Look at how all names are formatted in the file. It should be Gehrig, D., Scaramuzza, D.,  
Same comment for all new entries below.